### PR TITLE
Create rubocop-flexport repo for open sourcing

### PR DESCRIPTION
**Pull request by maxh**
Inspired by:

https://github.com/airbnb/ruby/tree/master/rubocop-airbnb

TODO: Figure out workflow for integrating with monorepo.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: ruby
+sudo: false
+rvm:
+  - 2.3
+  - 2.4
+  - 2.5
```
Comment (Gasparila) [inline .travis.yml]: Can we throw 2.6 on here too?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+custom formatting RuboCop rules. Our intention is generally to upstream all
+of the cops in the repository to the core RuboCop repo(s). But we maintain
+this repo as a place to open source our cops prior to their upstreaming.
+
+## Installation
+
+Add to your `Gemfile`:
+
+```
+gem 'rubocop-flexport'
```
Comment (Gasparila) [inline README.md]: Also mention including it in `.rubocop.yml`?

File: rubocop-flexport.gemspec [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  spec.email = ["rubocop@flexport.com"]
+  spec.homepage = "https://github.com/flexport/rubocop-flexport"
+  spec.license = "MIT"
+  spec.version = RuboCop::Flexport::VERSION
+  spec.platform = Gem::Platform::RUBY
+  spec.required_ruby_version = ">= 2.3"
+
+  spec.require_paths = ["lib"]
+  spec.files = Dir[
+    "{config,lib,spec}/**/*",
```
Comment (eric-zhu-uw) [inline rubocop-flexport.gemspec]: Might not need to include `spec/**/*` or `Gemfile` when open sourcing

**Comment by maxh** [comment]
I've concluded the better approach here is to just upstream our changes into RuboCop instead of open sourcing this repo. Closing.

**Pull request closed without merging**